### ISSUE TITLE
new BIOS options disappeared...

### ISSUE THREAD

**Issue by gomisensei** [libretro/mame2003-plus-libretro#251]
This may be due to the bi-directional options commits (which don't seem to be working at all, as far as i can tell), but the stv and neogeo/unibios options have disappeared

**Comment by markwkidd** [comment]
Whoops. I'll take a look.

There still needs to be one more bit of refactoring and a new function written in order to have the core be able to communicate changes in core options to the fontend once emulation starts -- and then we need some core options that actually benefit from this functionality. :)

I had been meaning for there to be no affect on functionality though in the meantime. I'll tag you once I figure out how to fix what I must have changed.

**Comment by gomisensei** [comment]
never mind, I was running the wrong executable,

**Comment by grant2258** [comment]
run with a --verbose and paste your log

**Comment by markwkidd** [comment]
I found the error :cry: 

Once I was done refactoring the core options there were "duplicate" default values for all the options that I got rid of with this commit: https://github.com/libretro/mame2003-plus-libretro/commit/d4a105d730b95d55dee523fb5a2bbde604203e66

That is, except I didn't get rid of `options.bios = NULL;`. The core option was setting the BIOS and then wiping it back to NULL afterward. I'll push a fix in a minute.
